libmamba/src/download/downloader.cpp

```diff
@@ -19,18 +19,28 @@
 #include "curl.hpp"
 #include "downloader_impl.hpp"
 
+// To find the path of `libmamba`'s loaded shared library.
+#if defined(__linux__) || __APPLE__ || __MACH__
+#include <dlfcn.h>
+#else
+#include <Windows.h>
+#endif
+
 namespace mamba::download
 {
     namespace
     {
 
-        constexpr std::array<const char*, 6> cert_locations{
+        constexpr std::array<const char*, 9> cert_locations{
             "/etc/ssl/certs/ca-certificates.crt",                 // Debian/Ubuntu/Gentoo etc.
             "/etc/pki/tls/certs/ca-bundle.crt",                   // Fedora/RHEL 6
             "/etc/ssl/ca-bundle.pem",                             // OpenSUSE
             "/etc/pki/tls/cacert.pem",                            // OpenELEC
             "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem",  // CentOS/RHEL 7
             "/etc/ssl/cert.pem",                                  // Alpine Linux
+            "/System/Library/Keychains/SystemRootCertificates.keychain",  // macOS
+            "/Library/Keychains/System.keychain",                         // macOS
+            "C:/Program Files/Common Files/SSL/certs/ca-bundle.crt"       // Windows
```

Comment:
Note for reviewers: I am searching for an authoritative list of all certificates on these platforms.

Reply:
How does the current version find the certificates on MacOS and Windows? Not saying we should not add this, but I want to avoid redundancy.

Reply:
I don't know. They might be found automatically on Windows and macOS.

I'll revert the change only to change the case of Linux.

Reply:
I eventually performed changes for all platforms.